log/README.md

```diff
@@ -65,58 +69,78 @@ import (
 func main() {
 	logger := kitlog.NewJSONLogger(os.Stdout)
 	stdlog.SetOutput(kitlog.NewStdlibAdapter(logger))
-	
 	stdlog.Print("I sure like pie")
 }
-```
 
-Output
+// Output:
+// {"msg":"I sure like pie","ts":"2016/01/01 12:34:56"}
 ```
-{"msg":"I sure like pie","ts":"2016/01/28 19:41:08"}
+
+Or, if, for legacy reasons,
+ you need to pipe all of your logging through the stdlib log package,
+ you can redirect Go kit logger to the stdlib logger.
+
+```go
+logger := kitlog.NewLogfmtLogger(kitlog.StdlibWriter{})
+logger.Log("legacy", true, "msg", "at least it's something")
+
+// Output:
+// 2016/01/01 12:34:56 legacy=true msg="at least it's something"
 ```
 
-Adding a timestamp to contextual logs
+### Timestamps and callers
 
 ```go
-func handle(logger log.Logger, req *Request) {
-	ctx := log.NewContext(logger).With("ts", log.DefaultTimestampUTC, "query", req.Query)
-	ctx.Log()
+var logger log.Logger
+logger = log.NewLogfmtLogger(os.Stderr)
+logger = log.NewContext(logger).With("ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
 
-	answer, err := process(ctx, req.Query)
-	if err != nil {
-		ctx.Log("err", err)
-		return
-	}
+logger.Log("msg", "hello")
 
-	ctx.Log("answer", answer)
-}
+// Output:
+// ts=2016-01-01T12:34:56Z caller=main.go:15 msg=hello
 ```
 
-Output
-```
-ts=2016-01-29T00:46:04Z query="some=query"
-ts=2016-01-29T00:46:04Z query="some=query" answer=42
-```
+## Supported output formats
 
-Adding caller info to contextual logs
+- [Logfmt](https://brandur.org/logfmt)
+- JSON
 
-```go
-func handle(logger log.Logger, req *Request) {
-	ctx := log.NewContext(logger).With("caller", log.DefaultCaller, "query", req.Query)
-	ctx.Log()
+## Enhancements
 
-	answer, err := process(ctx, req.Query)
-	if err != nil {
-		ctx.Log("err", err)
-		return
-	}
+`package log` is centered on the one-method Logger interface.
 
-	ctx.Log("answer", answer)
+```go
+type Logger interface {
+	Log(keyvals ...interface{}) error
 }
 ```
 
-Output
-```
-caller=logger.go:20 query="some=query"
-caller=logger.go:28 query="some=query" answer=42
-```
+This interface, and its supporting code like [log.Context](https://godoc.org/github.com/go-kit/kit/log#Context),
+ is the product of much iteration and evaluation.
+For more details on the evolution of the Logger interface,
+ see [The Hunt for a Logger Interface](http://go-talks.appspot.com/github.com/ChrisHines/talks/structured-logging/structured-logging.slide#1),
+ a talk by [Chris Hines](https://github.com/ChrisHines).
+Also, see
+ [#63](https://github.com/go-kit/kit/issues/63),
+ [#76](https://github.com/go-kit/kit/pull/76),
+ [#131](https://github.com/go-kit/kit/issues/131),
+ [#157](https://github.com/go-kit/kit/pull/157), and
+ [#252](https://github.com/go-kit/kit/pull/252)
```

Comment:
Please add #164 to this list. It explains why `Logger.Log` returns an error, and that seems to be a FAQ as well.